das-dashboard/backend/services/config_services.py

```
@@ -130,6 +113,35 @@ async def load_config(self, nested_config: dict) -> dict:
 
         return flat
 
+    async def sync_dashboard_config(self) -> list[dict]:
+        """Reload saved config and register it with das-cli for dashboard/metrics use."""
+        if os.path.exists(CONFIG_PATH):
```

Ruff (0.15.20)

[warning] Async functions should not use os.path methods, use trio.Path or anyio.path

(ASYNC240)